src/components/Fund/FundBanner.tsx

```diff
@@ -1,106 +1,161 @@
+                            >
+                                7D
+                            </button>
+                            <button
+                                className='px-4 py-3 bg-black text-white rounded-md text-xs cursor-pointer font-medium'
+                            >
+                                1M
+                            </button>
+                        </div>
+                    </div>
+                    <h2 className='text-white text-4xl mt-3'>${numberWithCommas(Number(portfolioValue))}</h2>
+                    <p className='text-white text-xs mt-3'>
+                        <Trans>
+                            Starting value: ${numberWithCommas(Number(startingValue))}
+                        </Trans>
+                    </p>
+                    <div className='flex justify-between items-center'>
+                        <div className='flex space-x-5 mt-3'>
+                            <p className='text-gray-400 text-xs'>
+                                <Trans>
+                                    Data updated: {dataUpdated} ago
+                                </Trans>
+                            </p>
+                            <button className='cursor-pointer'>
+                                <img src={updateNowIcon} alt="" />
+                            </button>
+                        </div>
+                        <div className='flex items-center space-x-6'>
+                            <p className='text-green-400 text-xs font-medium'>
+                                +${ numberWithCommas(Number(newlyAddedMoney)) }
+                            </p>
+                            <p
+                                className='text-green-400 text-xs font-medium bg-green-900 bg-opacity-70 py-1 px-2 rounded-2xl'
+                            >
+                                +{ upPercentage }%
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            {/* Portfolio, Trading, Yield Buttons */}
+            <div className='flex items-center justify-between mt-10'>
+                <div className='space-x-6'>
+                    <NavLink to='/fund/portfolio'>
+              <button onClick={() => setPty('portfolio')} className={`font-semibold text-xl shadow-md px-14 py-3 rounded-t-xl hover:bg-gray-700 ${pty === 'portfolio' ? 'text-yellow-400 bg-[#282835]' : 'text-white border-t border-x border-gray-700'}`}>
+                          <Trans>
+                              PORTFOLIO
+                          </Trans>
+                      </button>
+                    </NavLink>
+                    <NavLink to='/fund/trading'>
+                      <button onClick={() => setPty('trading')} className={`font-semibold text-xl shadow-md px-14 py-3 rounded-t-xl hover:bg-gray-700 ${pty === 'trading' ? 'text-yellow-400 bg-[#282835]' : 'text-white border-t border-x border-gray-700'}`}>
```

Comment:
this button needs to be a separate component so the style can be reused. pass a `selected` variable in and use that to highlight the style. it will be much cleaner that way

Reply:
Done